gordo_components/cli/cli.py

```diff
@@ -59,6 +59,8 @@ def gordo():
         format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
     )
 
+    logging.getLogger("matplotlib").setLevel(logging.INFO)
```

Comment:
Oh matplotlib... you devil. :smile:  Should this possibly be set to the global runtime level also?

Reply:
The "problem" by doing that is if we use the log_key and set it to debug, matplotlib will nag us with a wall of text. I think me and @epa095 talked about this and agreed to set it to INFO for now, otherwise it kind of destroys what we did in https://github.com/equinor/gordo-components/issues/657.
I think having another key for this is kind of unnecessary. I think we rather should turn it on manually if it ever gets a problem? But I am open to suggestions. @milesgranger @ryanjdillon